v0.7/medical_imaging/3d-unet/onnxruntime_SUT.py

```diff
@@ -0,0 +1,78 @@
+# coding=utf-8
+# Copyright (c) 2020 NVIDIA CORPORATION. All rights reserved.
+# Copyright 2020 Division of Medical Image Computing, German Cancer Research Center (DKFZ), Heidelberg, Germany
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import array
+import json
+import os
+import sys
+sys.path.insert(0, os.getcwd())
+
+import mlperf_loadgen as lg
+import numpy as np
+import onnxruntime
+from transformers import BertConfig, BertForQuestionAnswering
+from squad_QSL import get_squad_QSL
+
+class BERT_ONNXRuntime_SUT():
```

Comment:
I'm assuming we'll need to update this from BERT to 3D UNet. Shouldn't be difficult though using this as the template

Reply:
good catch!